src/options/options_handler.cpp

```diff
@@ -159,13 +159,13 @@ void OptionsHandler::checkBvSatSolver(std::string option, SatSolverMode m)
           || m == SatSolverMode::KISSAT))
   {
     if (options::bitblastMode() == options::BitblastMode::LAZY
-        && options::bitblastMode.wasSetByUser())
+        && Options::current().wasSetByUser(options::bitblastMode))
     {
       throwLazyBBUnsupported(m);
     }
-    if (!options::bitvectorToBool.wasSetByUser())
+    if (!Options::current().wasSetByUser(options::bitvectorToBool))
     {
-      options::bitvectorToBool.set(true);
+      Options::current().set(options::bitvectorToBool, true);
     }
   }
```

Comment:
Since we use this construct quite a lot:

```
if (!wasSetByUser())
{
  set(...)
}
```

How about we have an additional `Options::setDefault` for these cases that only sets an option if it was not set by the user?